packages/e2e-tests/utils/testData.ts

```diff
@@ -1,4 +1,10 @@
 export const testData = {
   village: "Dama",
-  villageID:"village-Dama"
+  villageID: "village-Dama",
+  department: "Cardiology",
+  departmentID: "department-Cardiology",
+  encounterType: "admission",
+  locationID: "location-EDBed1",
+  patientBillingTypeID: "patientType-Private",
+  patientID: undefined,
```

Comment:
This is standard casing for id at end 🙏 
```suggestion
  villageId: "village-Dama",
  department: "Cardiology",
  departmentId: "department-Cardiology",
  encounterType: "admission",
  locationId: "location-EDBed1",
  patientBillingTypeId: "patientType-Private",
  patientId: undefined,
```